jiant/trainer.py

```diff
@@ -860,7 +860,8 @@ def _calculate_validation_performance(
 
             all_val_metrics["%s_loss" % task.name] += loss.data.cpu().numpy()
             n_examples += get_output_attribute(out, "n_exs", self._cuda_device)
-            if print_output:
+            # The following doesn't work for other inputs, yet.
+            if print_output and "inputs" in batch and "words" in batch["inputs"]:
```

Comment:
This seems like a place where an assert would help: Right now, won't this cause jiant to just ignore a config option?

Reply:
I made it a NotImplementedError. Does that seem okay?

Reply:
Yep!